com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerModelLoader.cs

```diff
@@ -0,0 +1,87 @@
+using System.Collections.Generic;
+using UnityEngine;
+using System.Threading.Tasks;
+
+
+#if MROPENXR_PRESENT && (UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_ANDROID) && GLTFAST_PRESENT && KTX_PRESENT
+using Microsoft.MixedReality.OpenXR;
+using GLTFast;
```

Comment:
Are we only using GLTFast if we are also using the MR OpenXR plugin? Is there no path for gltf controller models without our plugin?

Reply:
For this controller model loader, the GLTFast code is only applicable for loading Controller models using the OpenXR plugin. In theory, there's nothing stopping us from using GLTFast to load a model from another source, but this class in it's current state is  solely focused on loading models via the MR OpenXR plugin's `ControllerModel` api